sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json

```diff
@@ -0,0 +1,3270 @@
+﻿{
```

Comment:
This doesn't belong in this PR

Reply:
oops this got included looks like. Will remove it.